benches/benches/benchmarks/resolve.rs

```diff
@@ -3,8 +3,8 @@ use ckb_app_config::{BlockAssemblerConfig, TxPoolConfig};
 use ckb_chain::chain::{ChainController, ChainService};
 use ckb_chain_spec::{ChainSpec, IssuedCell};
 use ckb_jsonrpc_types::JsonBytes;
-use ckb_launcher::SharedBuilder;
 use ckb_resource::Resource;
+use ckb_shared::SharedBuilder;
 use ckb_shared::{Shared, Snapshot};
```

Comment:
please merge into one line:
```suggestion
use ckb_shared::{Shared, SharedBuilder, Snapshot};
```

Reply:
There are multiple places need to merge these imports statements, going to push another commit to fix these.